### ISSUE TITLE
DateChooser on Mac Does Not Show Calendar

### ISSUE THREAD

**Issue by 3dp** [chriskiehl/Gooey#273]
When using the DateChooser on a Mac, I'm just getting a box with a date format and no actual calendar to pick from...

- OS:  OSX 10.3.3
- Python Version: 2.7.13
- Gooey Version: 1.0.0 (although I updated it yesterday)

I'm using `parser.add_argument('--importantDate',help="Pick your important date",widget="DateChooser")`
![screen shot 2018-03-07 at 4 16 43 pm](https://user-images.githubusercontent.com/1696050/37103793-fb434eee-2222-11e8-90e5-8d3e0de13ebe.png)

            `

**Comment by chriskiehl** [comment]
Just to be clear, this holds even when you click the box and/or toggle?

**Comment by 3dp** [comment]
Yes.  If I click on either of the up/down buttons it just changes the number that's selected (up or down).  No calendar pops up at all.

**Comment by jainraje** [comment]
I'm seeing the same behavior as the poster. No calendar pop-up.

**Comment by gregtronel** [comment]
Same issue on 10.12.6

**Comment by thecircleisround** [comment]
is there any fix to this?
